tests/system/action/motion/test_update.py

```diff
@@ -217,13 +217,14 @@ def test_update_correct_2(self) -> None:
             )
         self.assert_status_code(response, 200)
         model = self.get_model("motion/111")
-        assert model.get("state_extension") == "test_blablab_noon"
-        assert model.get("recommendation_extension") == "ext_sldennt [motion/112]"
+        assert model.get("state_extension") == "ext [motion/112] [motion/113]"
+        assert model.get("recommendation_extension") == "ext [motion/112] [motion/113]"
         assert model.get("category_id") == 4
         assert model.get("block_id") == 51
         assert model.get("supporter_ids") == []
         assert model.get("tag_ids") == []
         assert model.get("attachment_ids") == []
+        assert model.get("state_extension_reference_ids") == ["motion/112"]
```

Comment:
What's about motion/113?

Reply:
motion/113 does not exist and should therefore not be present in the relation. This test tests multiple things at once, maybe that's why it's a little confusing. Added a comment for this.